common/flatpak-transaction-log.c

```diff
@@ -0,0 +1,305 @@
+/* flatpak-transaction-log.c
```

Comment:
Nitpick: Do we have to include the filename in the header comment? It adds no value and is likely to be copy-pasted elsewhere and become incorrect.

Reply:
I'm not sure - it seems like this is a convention throughout flatpak generally? Full disclosure - I copypasted from `flatpak-db.c` here ;-)

Reply:
I guess @alexlarsson should answer this. Personally, I do not see any upsides to putting the filename at the top of the file.